### PR TITLE
hygiene(#268+): pr-preservation drain-log for #425 (CommonMark fence indent)

### PR DESCRIPTION
**Pull request by AceHack**
## Summary

Otto-268 follow-on: drain-log for PR #425 (post-merge cascade to #357 fence-detection logic). Captures one substantive parser-correctness fix per CommonMark §4.5.

Per Otto-250 (PR review comments + responses + resolutions are high-quality training signals).

## Coverage

- **`lstrip(' ')` + explicit tab-rejection on fence detection** (P1, parser correctness): parent #357's fence-detection used `raw_line.lstrip()` (silently consumes tabs) before checking the marker. Per CommonMark §4.5, code fences are recognized only when the opening marker is preceded by **at most 3 spaces** (not tabs). Fix: `lstrip(' ')` (space-only) + explicit tab-rejection. Tab-indented + 4+-space-indented fence-shaped lines correctly fail the marker check.

## Pattern observations

1. **CommonMark spec compliance** is its own findings class. Codex catches this class reliably when parser code is in a PR.
2. **`lstrip()` vs `lstrip(' ')`** is a subtle but load-bearing distinction in markdown parsing. CommonMark consistently distinguishes; many indent-related primitives need to be space-aware not whitespace-aware. Pre-commit-lint candidate: regex check on markdown-parser code for unconditional `.lstrip()` calls suggesting `' '` arg.
3. **Quiet-failure modes in markdown parsers are the most-dangerous bug class** — tab-indented fence-shaped lines were silently misclassified with no exception / warning / test failure. Pattern generalizes: any parser that silently misclassifies-vs-rejects on edge cases needs explicit reject-paths.

## Test plan

- [x] Finding preserved with severity + outcome class.
- [x] CommonMark §4.5 citation captured for future reference.
- [x] Drain-log shape matches existing `docs/pr-preservation/*-drain-log.md` template.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: docs/pr-preservation/425-drain-log.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+---
+
+## Threads
+
+### Thread 1 — `lstrip(' ')` + explicit tab-rejection on fence detection
+
+- Reviewer: chatgpt-codex-connector
+- Severity: P1 (parser correctness)
+- Finding: parent #357's fence-detection used `raw_line.lstrip()`
+  (which silently consumes tabs) before checking the marker. Per
```
[the remark this answers is not in the document]
Comment (AceHack) [reply docs/pr-preservation/425-drain-log.md:38]: Fixed in 808d8332e128bd55921da307af5d6ff51d3d4b08 via option (b) — dropped the 'full record' claim and reframed as 'abbreviated Otto-268-wave record' with explicit pointer to docs/pr-preservation/_patterns.md shape-divergence section and named contrast against canonical-shape examples (#108, #395). The shape-conformance gap is now documented honestly in #467 as a known divergence rather than a defect to bulk-rewrite.
